pulsar/transaction.go

```diff
@@ -17,7 +17,32 @@
 
 package pulsar
 
+import (
+	"context"
+)
+
+const (
+	Open = iota //The init state is open
```

Comment:
```suggestion
	Open = iota
```

Please add the API docs instead of just a simple comment.